### PR TITLE
Adding webdriver.js-example-project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

File: Node/webdriver.js_vanilla/tests/example.spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const AxeDevToolsWebdriverJS = require('@axe-devtools/webdriverjs');
+const WebDriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const Reporter = require('@axe-devtools/reporter').default;
+const async_hooks = require('async_hooks');
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/tests/example.spec.js:5]:
```suggestion
```

File: Node/webdriver.js_vanilla/tests/example.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// var driver = new WebDriver.Builder().forBrowser('chrome').build();
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/tests/example.spec.js:6]:
```suggestion
```

File: Node/webdriver.js_vanilla/tests/example.spec.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    // Building the report in different formats
+    await reporter.buildHTML('./a11y-results/html/');
+    await reporter.buildJUnitXML('./a11y-results/xml/');
+    await reporter.buildCSV('./a11y-results/csv/');
+
+    await driver.quit();
+}
+
+exampleTest()
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/tests/example.spec.js:38]: This is an async function and needs to be resolved via promise/await

File: Node/webdriver.js_vanilla/tests/example.spec.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+const AxeDevToolsWebdriverJS = require('@axe-devtools/webdriverjs');
+const WebDriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const Reporter = require('@axe-devtools/reporter').default;
+const async_hooks = require('async_hooks');
+// var driver = new WebDriver.Builder().forBrowser('chrome').build();
+
+
+
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/tests/example.spec.js:9]:
```suggestion
```

File: Node/webdriver.js_vanilla/package.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webdriver.js_vanilla",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "rimraf ./a11y-results && node ./tests/example.spec.js "
+  },
+  "keywords": [],
+  "author": "",
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/package.json:10]:
```suggestion
```

File: Node/webdriver.js_vanilla/package.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webdriver.js_vanilla",
+  "version": "1.0.0",
+  "description": "",
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/package.json:4]: We should add a description

File: Node/webdriver.js_vanilla/package.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "rimraf ./a11y-results && node ./tests/example.spec.js "
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```
Comment (michael-siek) [inline Node/webdriver.js_vanilla/package.json:12]: These should all be devdeps

**Review by michael-siek — CHANGES_REQUESTED** [review]

**Review by AdnoC — APPROVED** [review]

**Pull request merged**